fava/static/javascript/editor.js

```diff
@@ -53,11 +53,11 @@ function saveEditorContent(cm) {
       cm.focus();
       e.trigger('file-modified');
     }, () => {
-      e.trigger('error', `Saving ${fileName} failed.`);
+      e.trigger('error', window.favaTranslations.savingFailed.replace('$FILENAME', fileName));
```

Comment:
The helper function should also take care of the replacement. And I don't see a reason to change the variable syntax from `${}`.